tools/distpartitioning/data_shuffle.py

```diff
@@ -142,6 +146,27 @@ def gen_node_data(
 
         for local_part_id in range(num_parts // world_size):
             cond = node_partid_slice == (rank + local_part_id * world_size)
+
+        # No. of nodes in each process can differ significantly in lopsided distributions
+        # Synchronize on a per ntype basis
+        dist.barrier()
+
+        type_start, type_end = (
```

Comment:
are these 2 variables are used?

Reply:
This file was completely messed up during the merge conflict resolution. The same for-loop was repeated twice and with wrong indentation. So the CI was bound to fail. 

And to answer your question. Yes, these two variables are used in the for-loop. Now the code is cleaned up and you should be able to see these variables. clearly.